### PR TITLE
[DOCFIX] Add how to enable web/metrics on standby masters

### PR DESCRIPTION
**Pull request by jiacheliu3**
### What changes are proposed in this pull request?

Add docs for https://github.com/Alluxio/alluxio/pull/14259

### Why are the changes needed?

NA

### Does this PR introduce any user facing changes?

NA

### CODE REVIEW

File: docs/en/operation/Metrics-System.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ The following instances are currently supported:
 * Worker: The Alluxio worker process.
 * Client: Any process with the Alluxio client library.
 
+When the cluster is running with High Availability, by default the standby masters do not
+serve the metrics. From Alluxio 2.8 on, users can use `alluxio.standby.master.metrics.enabled=true`
+to enable the Web UI on the standby masters too.
```
Comment (Xenorith) [inline docs/en/operation/Metrics-System.md:28]:
```suggestion
serve metrics. Set `alluxio.standby.master.metrics.enabled=true`
to have the standby masters also serve metrics.
```

File: docs/en/operation/Web-Interface.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ The Alluxio master web interface can be found by visiting `http://<MASTER IP>:19
 For instance, if Alluxio was started locally, the master web interface
 can be viewed by visiting [localhost:19999](http://localhost:19999).
 
+When the cluster is running with High Availability, by default the standby masters
+do not serve the Web UI. From Alluxio 2.8 on, users can use `alluxio.standby.master.web.enabled=true`
+to enable the Web UI on the standby masters too.
```
Comment (Xenorith) [inline docs/en/operation/Web-Interface.md:24]:
```suggestion
do not serve the Web UI. Set `alluxio.standby.master.web.enabled=true`
to have the standby masters also serve the Web UI.
```

**Review by Xenorith — CHANGES_REQUESTED** [review]

File: docs/en/operation/Metrics-System.md [hunk 3]
```diff
@@ -23,6 +23,10 @@ The following instances are currently supported:
 * Worker: The Alluxio worker process.
 * Client: Any process with the Alluxio client library.
 
+When the cluster is running with High Availability, by default the standby masters do not
+serve the metrics. From Alluxio 2.8 on, users can use `alluxio.standby.master.metrics.enabled=true`
```
Comment (maobaolong) [inline docs/en/operation/Metrics-System.md:27]: https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/conf/PropertyKey.java#L6853

```
    public static final String STANDBY_MASTER_METRICS_SINK_ENABLED =
        "alluxio.standby.master.metrics.sink.enabled";
    public static final String STANDBY_MASTER_WEB_ENABLED =
        "alluxio.standby.master.web.enabled";
```

From the code related, the `alluxio.standby.master.metrics.enabled=true` seems to be changed to `alluxio.standby.master.metrics.sink.enabled`

**Review by maobaolong — APPROVED** [review]
It LGTM for this document improvement after correct the property key name. Thanks @jiacheliu3

File: docs/en/operation/Metrics-System.md (hunk 3 above)
Comment (jiacheliu3) [reply docs/en/operation/Metrics-System.md]: ooops

**Review by Xenorith — APPROVED** [review]

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
@Xenorith could you help cherry-picking this to 2.8? Thx!

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.8 please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.8 please

**Pull request merged**